packages/frontend/src/store.ts

```diff
@@ -222,7 +226,7 @@ export const defaultStore = markRaw(new Storage('base', {
 	},
 	animatedMfm: {
 		where: 'device',
-		default: false,
+		default: true,
```

Comment:
```suggestion
		default: window.matchMedia('(prefers-reduced-motion)').matches,
```

普通のブラウザではデフォルトON、動きを抑えたい設定にしてあるブラウザではデフォルトOFFにできるのでこっちのほうがベター

Reply:
強く同意

Reply:
👀